src/libbacktrace/macho.c

```diff
@@ -0,0 +1,1414 @@
+/* macho.c -- Get debug data from an Mach-O file for backtraces.
```

Comment:
Can you maybe insert a link to the pending patch in libbacktrace itself?

Clearly we're not in a good position to review this independently, but it'd be nice to be able to place this in the context of that project.